tests/e2e/testBatteries.spec.ts

```diff
@@ -0,0 +1,145 @@
+import { test } from "@playwright/test";
+import { loadExtension, delay } from "./helpers/loadExtension";
+
+/**
+ * "unit-name":{ // can be anything
+ *    url:"url to test", // the url of the page where the battery is supposed to work
+ *    cookieButtonSelector: {}, // optional: selector to accept the cookie notice, 
+ *                              // see clickButton() to know how to define a selector
+ *    output:{ // list of expected outputs to test against the battery's returned ln data 
+ *      recipient: '...', // check if recipient is equals to the provided value
+ *      description: {
+ *        startsWith: '...' // check if the description starts with the provided value.
+ *                          // This can be used to test for things that can't or aren't 
+ *                          // convenient to test in full. Eg. the icon url that can be a 
+ *                          // very long data url or contain session data
+ *      }
+ *      
+ *    }
+ * }
+ */
+const Units = {
+  "monetization": {
+    url: "https://getalby.com/",
+    output: {
+      recipient: 'hello@getalby.com',
+      name: "Alby",
+      description: "Alby brings Bitcoin to the web with in-browser payments and identity.",
+      icon: "https://getalby.com/website/_assets/alby_icon_head_icon-ICVYH45J.png"
+    }
+  }
+}
+
+
+/**
+ * Try to click a button if exists until timeout is reached
+ * @param page the page object 
+ * @param selectionData either a string representing a selector or an object structured like this 
+ *      {
+ *        $: "selector", // optional, "button" will be used if not specified 
+ *        innerText: "text content of the button", // optional
+ *        waitNavigator: true, // wait for page to be idle after clicking
+ *        scrollDown: true // scroll the page down before clicking (this is used eg. for the cookies notice)
+ *      }
+ */
+async function clickButton(page, selectionData) {
+  const selector = typeof selectionData == "object" && selectionData.$ ? selectionData.$ : "button";
+  const start = Date.now();
+  return new Promise<void>((rr) => {
+    const findAndClickButton = async () => {
+      const buttons = await page.$$(selector);
+      for (const button of buttons) {
+        const buttonText = await page.evaluate(el => el.textContent, button);
+        if (buttonText && selectionData.innerText && buttonText.indexOf(selectionData.innerText) == -1) continue;
+        if (selectionData.scrollDown) await page.evaluate(() => window.scrollBy(0, window.innerHeight));
+        await delay(100);
+        await button.click();
+        if (selectionData.waitNavigator) {
+          await page.waitForNavigation({ waitUntil: 'networkidle2' })
+        } else {
+          await delay(100);
+        }
+        rr();
+        return;
+      }
+      console.log("Looking for button ...", selectionData);
```

Comment:
Better use `.info` instead og `.log`. Log always looks like forgotten debug code.  
Same for other usages.

Reply:
should be resolved